### ISSUE TITLE
Wrong "platform" parameter passed to "make"

### ISSUE THREAD

**Issue by obert01** [libretro/libretro-super#139]
Wrong "platform" parameter passed to "make"

Hello,

I am using libretro-super on a Linux ARM box (odroid) to compile various cores. 

I have the impression that the shell script does not pass the right parameters to "make" for building cores. 

When I try to build libretro_mame for instance, the shell script issues the following command:
make -f Makefile.libretro TARGET=mame platform=unix-neon-cortexa9-hardfloat ...

But the makefile seems to expect something like:
make -f Makefile.libretro TARGET=mame platform=armv-unix-neon-cortexa9-hardfloat ...

In other words, "armv" does not appear in the "platform" string passed as an argument to "make". It is a problem because the compilation starts with standard UNIX settings, not with the ARM-specific ones. 

I have the impression that the issue is due to the way the variable $FORMAT_COMPILER_TARGET is constructed in libretro-config.sh. But I don't know how to fix it properly. 

Can someone propose a fix or tell me what I am doin wrong with libretro-super?

Thanks a lot.
## 

Olivier

**Comment by paulguy** [comment]
I have a similar problem building on raspberry pi 2.  Enabling hardfloat or neon in libretro-config.sh appends -hardfloat and -neon to the platform variable, however most of the makefiles only detect for if platform = unix, and revert to some braindead windows defaults otherwise, completely breaking the build.
